### ISSUE TITLE
[Feature Request] Chromecast support for downloaded movies and tv shows

### ISSUE THREAD

**Issue by 32423d** [LagradOst/CloudStream-3#1192]
### Describe your suggested feature

Currently there are issues with tv shows and movies not casting properly due to issues with the providers themselves. Since the issues are with the providers themselves I propose adding Chromecast support for downloaded movies and tv show of providers. The Chromecast issues with providers will most likely never be solved because the issue would need to be fixed on the providers end but if Chromecast support was added for downloaded movies and tv shows of providers users would easily be able to cast all their cloudstream content for now on (assuming the feature is maintained). 

### Other details

VLC's android app, another video player for android, currently supports casting downloaded video files but there is currently no subtitle support. Since subtitle files are downloaded alongside the downloads of tv shows and movies in cloudstream it would be nice if there would also be subtitle support if this feature was to be implemented.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by LagradOst** [comment]
No, chromecasting downloaded files is not as easy as isEnabled=true
